src/client/SyntheticClient.h

```diff
@@ -150,7 +150,7 @@ class SyntheticClient {
       r += cwd.last_dentry().c_str()[0];                                         // slightly permuted
     r %= contents.size();
 
-    map<string,struct stat*>::iterator it = contents.begin();
+    std::map<std::string,struct stat*>::iterator it = contents.begin();
```

Comment:
auto?

Reply:
there are lots of places like this. i did a search-n-replace in most cases. this would be simpler and less error-prune. please feel free to create a follow-up change to use `auto` when appropriate.